src/lib/Libutil/pbs_aes_encrypt.c

```diff
@@ -237,6 +242,10 @@ decode_from_base64(char* buffer, unsigned char** ret_decoded_data, size_t* ret_d
 
 	mem_obj1 = BIO_new_mem_buf(buffer, -1);
```

Comment:
both BIO_new_mem_buf() and BIO_push() returns pointers, can they return NULL? Should we check for failures of those  and similar such API calls everywhere as well?

Reply:
i checked these calls but they do not return NULL

Reply:
Do you know any doc that says it won't return NULL or won't fail?